api/SVGViewElement.json

```diff
@@ -148,34 +148,34 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": true,
+              "version_added": "1",
               "version_removed": "56"
             },
             "chrome_android": {
-              "version_added": true,
+              "version_added": "18",
               "version_removed": "56"
             },
             "edge": {
               "version_added": "12",
               "version_removed": "79"
             },
             "firefox": {
-              "version_added": true,
+              "version_added": "15",
```

Comment:
Can you back out these Firefox changes, or update the PR title and description to link to evidence for them?

Reply:
Updated!  The Firefox data comes from the collector.